docs/api/web-contents.md

```diff
@@ -777,10 +777,17 @@ Emitted when a `<webview>` has been attached to this web contents.
 Returns:
 
 * `event` Event
-* `level` Integer - The log level, from 0 to 3. In order it matches `verbose`, `info`, `warning` and `error`.
-* `message` string - The actual console message
-* `line` Integer - The line number of the source that triggered this console message
-* `sourceId` string
+* `_` never - deprecated parameter
```

Comment:
Kinda painful that there's not a safe way to eventually remove these deprecated parameters :(

Have you considered adding a new event like `console-message-details` or something that just has the `event` and `details` parameters? That way we could mark `console-message` as deprecated and have a path for eventually cleaning it up

Reply:
I'd rather not duplicate the event as that results in double the gin conversions, double the JS processing, etc.  It all adds up

Reply:
Would it be expensive to always add a listener which optionally emits for the older event? This would avoid the double gin conversion.
```ts
// lib/browser/api/web-contents.ts
WebContents.prototype._init = function () {
  this.on('console-message-added', (e, details) => {
    if (this.listenerCount('console-message') > 0) {
      this.emit('console-message', e, details.level, details.message, details.line, details.sourceId);
    }
  });
}
```

Regarding naming, maybe we can consider using "console-message-added" which lines up closely to [`Console.messageAdded`](https://chromedevtools.github.io/devtools-protocol/tot/Console/#event-messageAdded) in CDP.